### PR TITLE
enable open-cluster-management registration integration test

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,3 +17,11 @@ clean:
 .PHONY: clean
 
 GO_TEST_PACKAGES :=./pkg/... ./cmd/...
+
+test-install-ginkgo:
+	go get github.com/onsi/ginkgo/ginkgo
```
Comment (pmorie) [inline Makefile:22]: You don't need to pull the ginkgo binary to run the ginkgo tests, instead build the tests with `-c` and you will get a test binary that you can run the exact same way.

File: build/build-image/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM openshift/origin-release:golang-1.13
```
Comment (pmorie) [inline build/build-image/Dockerfile:1]: It's not clear yet what this Dockerfile is for - why do we need it?

File: Makefile (hunk 1 above)
Comment (skeeey) [reply Makefile:22]: cool, using `go test -c`

File: build/build-image/Dockerfile (hunk 2 above)
Comment (skeeey) [reply build/build-image/Dockerfile:1]: I try to prepare the integration test env (download kubebuilder for kube-apiserver and etcd) with this Dockerfile, I can use make to replace it, thus we don't need this Dockerfile

**Pull request closed without merging**